website/content/en/preview/AWS/provisioning.md

```diff
@@ -150,6 +150,21 @@ spec:
       httpTokens: required
 ```
 
+### Amazon Machine Image (AMI) Family
+
+The AMI used when provisioning nodes can be controlled by the `amiFamily` field. Based on the value set for `amiFamily`, Karpenter will automatically query for the appropriate [EKS optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-amis.html) via AWS Systems Manager (SSM). 
+
+Currently, Karpenter supports `amiFamily` values `al2`, `bottlerocket`, and `ubuntu`. GPUs are only supported with `al2`.
+
+Note: If a custom launch template is specified, then the AMI value in the launch template is used rather than the `amiFamily` value.

```

Comment:
Should we include this in validation?

Reply:
That's a good idea. We should include the other fields like this as well (securityGroupSelector, InstanceProfileName, MetadataOptipns, etc)

Reply:
Ditto, see my comment below along the same lines